applanix_bridge/launch/applanix_bridge.launch

```diff
@@ -3,7 +3,7 @@
   <arg name="publish_tf" default="True"/>
   <arg name="odom_frame" default="gps_odom"/>
   <arg name="base_frame" default="base_link"/>
-  <arg name="zero_start" default="True"/>
```

Comment:
The previous behaviour was to either start at zero, or report location as easting/northing values absolute within the UTM zone.

The new behaviour is to either start at zero, or use the specified easting/northing at the offset. You can recover the previous behaviour by an origin of [0,0,0], but I'm wondering if that should be made more obvious.

Is it worth preserving the behaviour of the existing `zero_start`? Does the following do the right thing in roslaunch?

```
<!-- By default use the first GPS fix as an origin point. Set zero_start to False
     to instead use the UTM origin. Alternatively, set origin to { easting: X, northing:
     Y, altitude: Z } to specific a UTM-relative local origin. -->
<arg name="zero_start" default="True" />
<arg name="origin" default="None" if="$(arg zero_start)" />
<arg name="origin" default="[0,0,0]" unless="$(arg zero_start)" />
```